kernel/src/debug.rs

```diff
@@ -183,7 +183,7 @@ macro_rules! debug_gpio {
 // debug! and debug_verbose! support
 
 pub const APPID_IDX: usize = 255;
-const BUF_SIZE: usize = 1024;
+const BUF_SIZE: usize = 20000;
 
```

Comment:
revert `BUF_SIZE`, then I'm pretty sure that `nrf51dk` will build correctly!

Reply:
Yeah, that fixed it - thanks!

Reply:
Just a note, with the BUF_SIZE at 1024, the kernel will panic if the 6lowpan test case is run as it is currently designed and an incorrect packet is received. In this case, the current debug output will print the expected packet and the received packet, which overflows the debug buffer.

Reply:
All right, I think conditional compilation via `features` could be solution to change `BUF_SIZE` between different builds!